### PR TITLE
[sdarq][frontend] Clean dependencies

### PR DESCRIPTION
**Pull request by zbedo**
This PR:

- Cleans unused dependencies
- Fixes possible information exposure through an exception

### CODE REVIEW

File: sdarq/backend/src/app.py
Comment (TomConner) [inline sdarq/backend/src/app.py:1]: Good not to put stack trace in response, but why not log the stack trace to allow debugging?

**Review by TomConner — CHANGES_REQUESTED** [review]
I have no issue with the dependency updates but I'd hate to see stack traces lost when an exception is thrown (even though good to keep them out of the HTTP response).

File: sdarq/backend/src/app.py
Comment (zbedo) [reply sdarq/backend/src/app.py:1]: Yes, it is logged here https://github.com/broadinstitute/dsp-appsec-infrastructure-apps/pull/466/files#diff-f3cf3becb9b27ba37e46c4ad5721f858547354b815881c215a7c0a091dfc4543R494-R495 and https://github.com/broadinstitute/dsp-appsec-infrastructure-apps/pull/466/files#diff-f3cf3becb9b27ba37e46c4ad5721f858547354b815881c215a7c0a091dfc4543R815-R816

Comment (zbedo) [reply sdarq/backend/src/app.py:1]: Just removed them from HTTP response but they are logged

Comment (TomConner) [reply sdarq/backend/src/app.py:1]: Oops, yes, the exception message is logged and stack traces were never in play.

**Review by TomConner — APPROVED** [review]

**Pull request merged**